.github/workflows/mirror-php-package.yml

```diff
@@ -0,0 +1,38 @@
+name: Mirror PHP package for release
+on:
+    workflow_dispatch:
+        inputs:
+            package:
+                description: 'Enter package name to mirror (e.g. @woocommerce/email-editor-config). It must refer to a name defined in the package.json file of the package in the packages/php directory.'
+                required: true
+
+permissions: {}
+
+jobs:
+    mirror:
+        name: Run packages release script
+        runs-on: ubuntu-latest
+        permissions:
+            contents: read
+        steps:
+            - name: Checkout monorepo
+              uses: actions/checkout@v4
+
+            - name: Setup WooCommerce Monorepo
+              uses: ./.github/actions/setup-woocommerce-monorepo
+            
+            - name: Build package for mirroring
+              run: |
+                  pnpm --filter "${{ github.event.inputs.package }}" build:composer-package
+
+            - name: Grab package build directory
+              run: echo "mirror_dir=$( pnpm --filter ${{ github.event.inputs.package }} list --json | jq --compact-output '.[]' | jq --raw-output '( .path + "/build")' )" >> $GITHUB_ENV
```

Comment:
I think this belongs to the previous step: if we generated the sources for mirroring there, the path export belongs there.